www/src/app/components/report/UserReport.js

```diff
@@ -22,17 +23,17 @@ export default function UserReport(props) {
 
         <h3 className="tile__subtitle">Goals for this week</h3>
         <ul className="tile__updates">
-          {props.goals.map(up => <Update key={up._id}>{up.text}</Update>)}
+          {props.goals.map(up => <Update key={up._id}>{linkify(up.text)}</Update>)}
```

Comment:
it's fine to have the helper itself in `utils` but I'd prefer to call inside of the `Update` components rather than in the caller code.